src/screens/DataApi/DataCoverageTable.style.tsx

```diff
@@ -0,0 +1,77 @@
+import { TableHead, Typography } from '@material-ui/core';
+import TableCell from '@material-ui/core/TableCell';
+import TableRow from '@material-ui/core/TableRow';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+import { InfoIcon } from 'components/InfoTooltip/Tooltip.style';
+import styled from 'styled-components';
+
+export const TextWithTooltipContainer = styled.div`
+  ${InfoIcon} {
+    margin-left: 0.2rem;
+    vertical-align: middle;
+  }
+`;
+
+export const UnderlinedText = styled(Typography)`
+  text-decoration-line: underline;
+  text-decoration-style: dotted;
+  text-underline-position: under;
+  vertical-align: middle;
+  font-size: 14px;
+`;
```

Comment:
Design feedback: The dotted underline makes me think that I would see the tooltip by hovering on the text, but when I do nothing happens. Maybe we should make the tooltip show up when hovering the text or the icon?

![image](https://user-images.githubusercontent.com/114084/110869546-86441b80-827f-11eb-8d86-bd9ee2877f03.png)